modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -178,16 +178,23 @@ static void addTensorRTExecutionProvider(Ort::SessionOptions *session_options,
 
 static void addOpenVINOExecutionProvider(Ort::SessionOptions *session_options,
                                          const cv::gapi::onnx::ep::OpenVINO &ov_ep) {
-     OrtOpenVINOProviderOptions options{};
-     options.device_type = ov_ep.device_type.c_str();
-     options.cache_dir = ov_ep.cache_dir.c_str();
-     options.num_of_threads = ov_ep.num_of_threads;
-     options.enable_opencl_throttling = ov_ep.enable_opencl_throttling;
-     options.enable_dynamic_shapes = ov_ep.enable_dynamic_shapes;
-     options.context = nullptr;
-
+     std::unordered_map<std::string, std::string> options = ov_ep.ParamsMap;
+    
+     auto checkAndAddOption = [&options](const std::string& key, const std::string& value) {
+        if (value.empty()) return;
```

Comment:
You need to check if value is default or not. E.g:

In case "num_of_threads" the default value is 0, when you convert it to string it will be "0" - which is not empty but still default.

This code is perfectly fine, but will fail because of collision
```
ovep.cfgParamsMap({{"num_of_threads", "10"}});
```